src/pre_defined_classes/mathematical_program/hard/q4/CalcBill.java

```diff
@@ -0,0 +1,23 @@
+package pre_defined_classes.mathematical_program.hard.q4;
+
+/*
+Make a Java program that determines the cost of some take-out sushi, given that you tip at 15% and want to round your
```

Comment:
Maybe we should add that the cost of the sushi is 23.75 to the question

Reply:
As this variable is just called total, which I think is a bit confusing

Reply:
I have made some modifications to the variable names and question description which should make it look clearer. I'd appreciate it if you could look it over to see if it makes more sense now.

Reply:
I think this is much better now

Reply:
Did you update the document with this as well?